data/base-town.yaml

```diff
@@ -199,6 +199,14 @@ Shard:
     inner_room:
   almsbox:
     id: 13143
+  guild_leaders:
```

Comment:
Would you be up for including the entire list for each city (similar to https://github.com/rpherbig/dr-scripts/commit/d82da1536095c75fb23c1428896efbb516eab40e#diff-e65384bcbc952bdd340d0bf9c06b452cR84) so we can see at a glance which ones are missing?

Reply:
Done, also made leth use crossing guildleaders and boar clan use the same as hib ones.